src/core/action.ts

```diff
@@ -28,6 +28,14 @@ export type IMiddleWareEvent = {
 let nextActionId = 1
 let currentActionContext: IMiddleWareEvent | null = null
 
+export function getSelf(): IStateTreeNode {
+    if (!currentActionContext || !currentActionContext.context)
```

Comment:
initializer functions (`.views` / `.actions`) don't create an action context. How do we make them return the correct self?